### PR TITLE
ESSIFI-168

### PR DESCRIPTION
**Pull request by zoemaas**
- Updated the dependencies to use the latest versions of pex and pex-models;
   - Updated the code;

### CODE REVIEW

File: src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class PresentationDefinitionWrapperEntity {
 
   @Column()
   // @ts-ignore
-  presentation_definition: PresentationDefinition;
+  presentation_definition: PresentationDefinitionV1 | PresentationDefinitionV2;
```
Comment (nklomp) [inline src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts]: Suggest to update dependency to @sphereon/pex 0.6.3-unstable.1 as that contains an `IPresentationDefinition` type which is essentially PresentationDefinitionV1 | PresentationDefinitionV2

**Review by nklomp — CHANGES_REQUESTED** [review]
Suggest to update dependency to @sphereon/pex 0.6.3-unstable.1 as that contains an IPresentationDefinition type which is essentially PresentationDefinitionV1 | PresentationDefinitionV2

**Review by sksadjad — APPROVED** [review]
looks good.

File: src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts (hunk 1 above)
Comment (zoemaas) [reply src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts]: Fixed!

**Review by nklomp — APPROVED** [review]

**Pull request merged**